data/commercial_building_benchmarks/Unmatched.txt

```diff
@@ -1,19 +1,238 @@
+SHOWROOM
+TOURIST OFFICE
+BEAUTY SALON / MASSAGE
+MOTOR ACCESSORIES
+CLOTHES SHOP
+JEWELLERS
+COFFEE SHOP
+ELECTRICAL / ELECTRONIC
+HARDWARE / DIY
+SURGERY
+ETHNIC FOOD SHOP
+DISCOUNT STORE
+CYCLE SHOP
+OFFICE SUPPLIES
+FASHION ACCESSORY
+FURNISHINGS
+AUCTIONEER
+CREDIT UNION
+FLORIST
+PHOTO PROCESSING SHOP
+MISCELLANEOUS
+OFFICE
+CRAFT SHOP
+CHARITY SHOP
+TATTOO PARLOUR
+OPTICIAN
+BRIDAL / FORMAL WEAR
+ADVENTURE / ARMY / CAMPING
+POST OFFICE
+INDIAN TAKE AWAY
+SANDWICH / JUICE BAR
+FURNITURE
+PHONE SHOP
+SHOE REPAIR / KEY CUT
+BUTCHERS / FISH MONGERS
+GREENGROCER
+DELICATESSEN
+MUSIC-INSTRUMENTS
+LIGHTING / LAMP
+VIDEO SHOP
+COSMETIC SHOP
+GAME SHOP
+ANTIQUE SHOP
+MARKET
+TAXI OFFICE
+ADULT SHOP
+TRAVEL AGENCY
+INTERNET CAFE
+CARD / STATIONERY / PRINT
+MUSIC-RECORDS / DVDS / VIDEOS
+COMPUTER SHOP
+TAILORING
+HEALTH FOOD SHOP
+RESTAURANT (DRIVE THRU)
+KIOSK
+BUILDING SOCIETY
+GARDEN SHOP
 nan
-MORGUE
-EASEMENT OF...
-NO USE SELECTED
-SHELTER
-ARMY BARRACKS
-STUDIO
-RIGHT OF LAYING PIPES
-TRANSFORMER STATION
-
-PLANT/OTHER - PIPELINES
-GAS WORKS
-GAS INSTALLATION
+PET SHOP
+BANK
+HOUSEHOLD GOODS
+NURSERY (MOTHERCARE)
+TOY SHOP
+TILE SHOP
+VEHICLE HIRE
+PEN SHOP
+MOTOR SHOWROOM
+DIY SUPERSTORE
+HARDWARE
+DISCOUNT
+COMPUTER
+SHOWROOM (INDUSTRIAL)
+WORKSHOP
+WAREHOUSE
+SHOWROOMS / OFFICES
+OFFICES (BUSINESS PARK)
+SECURITY BUILDING
+PRINTING WORKS
+FOOD PREPARATION
+BAKERY
+FACTORY
+PROVENDER MILL / FLOUR MILL
+BUS DEPOT
+CONCRETE WORKS
+FACTORY (MEAT)
+FACTORY (FISH)
+BULK STORES
+WAREHOUSE / WAREROOMS
+WAREHOUSE CASH &CARRY
+WAREHOUSE (HIGH BAY > 10M)
+SORTING OFFICE
+COLD STORE
+ASHPHALT PLANT
+DISTRIBUTION CENTRE
+BUS SHELTER
+DANCE STUDIO
+HANDBALL ALLEY
+HERITAGE / INTERPRETATIVE CENTRE
+HALL
+OIL / FUEL DEPOT
+PORT
+BUS STATION
+BIKE STATIONS
+OFFICE (OTHER)
+OFFICE (OFFICE PARK)
+ABATTOIR
+PUMPING STATION
+STADIUM (SOCCER)
+COMMUNITY HALL
+STADIUM (RUGBY)
+STADIUM (OTHER)
+CLUB HOUSE (HOCKEY)
+STADIUM (GAA)
+APART / HOTEL
+CONFERENCE CENTRE
 AVIATION FUEL DEPOT
 MOTOR FUEL SALES
 MOTORWAY SERVICE STATION
-LANDFILL
-TURBARY
+CAR VALET AREA (NO BUILDINGS)
+CAR PARK (INDUSTRIAL)
+GARDA STATION
+SURGERY (HOUSE)
+SURGERY (OFFICE)
+CLINIC
+SURGERY (RETAIL)
+HEALTH CENTRE
+MEDICAL CENTRE
+SURGERY (PURPOSE BUILT)
+SURGERY (OPTICIAN)
+HOSPITAL
+SURGERY (PHYSIOTHERAPIST)
+SURGERY (OTHER)
+VAN DEALERSHIP
+WAREHOUSE (BONDED)
+WEIGHBRIDGE
+OFFICE (GEORGIAN / VICTORIAN)
+OFFICE (2ND GENERATION)
+OFFICE (3RD GENERATION)
+OFFICE (OVER THE SHOP)
+OFFICE (INDUSTRIAL)
+OFFICE (OWN DOOR)
+OFFICE (HOUSE)
+STUDIO
+OFFICE (MEWS)
+OFFICE (PURPOSE BUILT PRE 1960)
+OFFICE (SURGERY)
+OFFICE (1ST GENERATION)
+CAR PARK
+OFFICE (BUSINESS PARK)
+OFFICE (EMBASSY)
+OFFICE (SECURITY BUILDING)
+HOLIDAY COMPLEX
+TELEPHONE (EXCHANGE)
+TILE
+SUPERMARKET
+NURSERY  (MOTHERCARE)
+FIREPLACES
+RETAIL WAREHOUSE
+CARAVAN PARK
+OTHER
+HOUSE
+HEALTH FARM
+DENTAL WORKSHOP
+GARDEN CENTRE
+COMMERCIAL GALLERY
+HOSTEL
+WATER
+BINGO HALL
+SQUASH COURT
+FISHERY
+QUAY / JETTY
+PASTORAL CENTER
```

Comment:
Revert to previous 'Unmatched.txt'!  When the benchmarks were updated these unmatched vo uses disappeared